File: src/main/java/com/p2p/lending/service/CreditlimitService.java
package com.p2p.lending.service;

import java.util.List;
import java.util.Map;

import com.p2p.lending.pojo.Creditlimit;

public interface CreditlimitService {

	public List<Creditlimit> queryCreditlimits(Map<String, Object> map);
	public int updateCreditlimit(Map<String, Object> map);
	public int insertCreditlimit(Map<String, Object> map);

}

File: src/main/java/com/p2p/lending/service/CreditlimitService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.Creditlimit

interface CreditlimitService {

    fun queryCreditlimits(map: Map<String, Any>): List<Creditlimit>
    fun updateCreditlimit(map: Map<String, Any>): Int
    fun insertCreditlimit(map: Map<String, Any>): Int

}
